### PR TITLE
ytfzf: init at 1.0.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115340` at 49538473 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/a8398b4527f510ddfacb54a787d27159">ytfzf</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Please add a comment on the line above, explaining the purpose of this patch.
Near pkgs/tools/misc/ytfzf/default.nix:21:5:
```
   |
21 |     (substituteAll {
   |     ^
```
</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
```
results/ytfzf/bin/ytfzf --version
youtube-dl not found. Please install it.
```

Please wrap youtube-dl.

**Comment by dotlambda** [comment]
I decided to use `makeWrapper` instead of patching in absolute paths. That also makes updating easier.

File: pkgs/tools/misc/ytfzf/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  src = fetchFromGitHub {
+    owner = "pystardust";
+    repo = "ytfzf";
+    rev = "v${version}";
+    sha256 = "09znixn8mpkxipv2x3nrfxr2i8g7y58v25qssqf092j9lh85sf9h";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/ytfzf/default.nix:26]:
```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**